.gitmodules

```diff
@@ -17,19 +13,6 @@
 	path = core/src/main/jni/redsocks
 	url = https://github.com/shadowsocks/redsocks.git
 	branch = shadowsocks-android
-[submodule "core/src/main/jni/mbedtls"]
-	path = core/src/main/jni/mbedtls
-	url = https://github.com/ARMmbed/mbedtls
-[submodule "core/src/main/jni/pcre"]
-	path = core/src/main/jni/pcre
-	url = https://android.googlesource.com/platform/external/pcre
-[submodule "core/src/main/jni/libsodium"]
-	path = core/src/main/jni/libsodium
-	url = https://github.com/jedisct1/libsodium.git
-	branch = stable
-[submodule "core/src/main/jni/libev"]
-	path = core/src/main/jni/libev
-	url = https://git.lighttpd.net/libev.git
-[submodule "core/src/main/jni/re2"]
-	path = core/src/main/jni/re2
-	url = https://github.com/google/re2.git
+[submodule "core/src/main/rust/shadowsocks-rust"]
+	path = core/src/main/rust/shadowsocks-rust
+	url = https://github.com/madeye/shadowsocks-rust
```

Comment:
Should this be changed to `shadowsocks/shadowsocks-rust`?

Reply:
shadowsocks/shadowsocks-rust#211 is not merged yet.

After that, we should change the submodule to upstream.

Reply:
As I mentioned I think that PR doesn't seem too important for this PR right now. :rofl: